pnpm-lock.yaml

```diff
@@ -192,13 +192,13 @@ importers:
       '@emotion/eslint-plugin': 11.11.0
       '@emotion/react': 11.11.1_tznpbkknyqq5lcv5sgrl332jvu
       '@guardian/ab-core': link:../ab-core
-      '@guardian/commercial': 10.0.0_typescript@5.1.3
-      '@guardian/consent-management-platform': 13.5.0_@guardian+libs@15.1.0
+      '@guardian/commercial': 10.8.0_typescript@5.1.3
+      '@guardian/consent-management-platform': 13.6.1_uo3bjqllavv5kwoq2zqgzcz5h4
       '@guardian/eslint-plugin-source-foundations': link:../eslint-plugin-source-foundations
       '@guardian/eslint-plugin-source-react-components': link:../eslint-plugin-source-react-components
-      '@guardian/libs': 15.1.0_xfreuenalcno2zxheqz32kfzfe
-      '@guardian/source-foundations': 12.0.0_xfreuenalcno2zxheqz32kfzfe
-      '@guardian/source-react-components': 15.0.1_klugjjbb25o62o6wn2a7ow2ku4
+      '@guardian/libs': link:../libs
+      '@guardian/source-foundations': link:../source-foundations
+      '@guardian/source-react-components': link:../source-react-components
```

Comment:
Just out of curiosity, what does this change do?

Reply:
This looks like it might be incorrect, as packages should point to an immutable store, not a locally linked package?

Reply:
my hunch is that's pnpm knowing what it has locally and preferring it over remote?